toolium/behave/env_utils.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) Telefonica Digital.
+# ElevenPaths QA Team <qateam@11paths.com>
```

Comment:
Toolium has Apache License. Use the same copyright header as the rest of the files.

Reply:
Fixed in 5fd6758